.github/workflows/codeql.yml

```diff
@@ -0,0 +1,86 @@
+name: "CodeQL"
```

Comment:
This file has been edited across multiple commits with instances of code being removed and then re-added. To streamline the review, the offending commits should be removed, either by rebasing or squashing them into a single, meaningful commit.